zio-http-example/src/main/scala/example/SSEServer.scala

```diff
@@ -0,0 +1,25 @@
+package example
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter.ISO_LOCAL_TIME
+
+import zio.{ExitCode, Schedule, URIO, ZIOAppDefault, durationInt}
+
+import zio.stream.ZStream
+
+import zio.http._
+import zio.http.sse.ServerSentEvent
```

Comment:
Just one more quick comment here: if there's only ever going to be one thing in `sse`, then the package doesn't really pull its weight, and we should consider moving its sole data type to `zio.http`. What do you think?

Reply:
I was thinking that maybe we have to provide some encoding/decoding support for Server Sent Event, but I guess we can move it for now in `zio.http` and update it later if is need it.